pkg/storage/engine/rocksdb/db.cc

```diff
@@ -1404,12 +1404,22 @@ DBString DBEngine::GetUserProperties() {
 
     auto ts_min = userprops.find("crdb.ts.min");
     if (ts_min != userprops.end()) {
-      DecodeHLCTimestamp(rocksdb::Slice(ts_min->second), sst->mutable_ts_min());
+      if (!DecodeHLCTimestamp(rocksdb::Slice(ts_min->second), sst->mutable_ts_min())) {
+        google::protobuf::SStringPrintf(all.mutable_error(),
+              "unable to decode crdb.ts.min value '%s' in table %s",
+              ts_min->second.c_str(), sst->path().c_str());
```

Comment:
Maybe format the timestamp as hex. I think something like `rocksdb::Slice(ts_min->second).ToString(true).c_str()`.